### PR TITLE
Service: ePaper

### PR DESCRIPTION
**Pull request by pelikhan**
This pull request adds a new service definition for Jacdac.

### CODE REVIEW

File: services/epaper.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    const variant: Variant @ variant
+
+The type of screen and number of bits per pixels. Monochrome = 1bit, Grayscale4 = 2bits, Tricolor = 2bits.
+
+## Commands
+
+    command paint @ 0x80 {
+        colors: pipe
+    }
+    pipe report colors {
```
Comment (pelikhan) [inline services/epaper.md:35]: this should be the other way around, it should be a pipe towards the display

Comment (mmoskal) [reply services/epaper.md:35]: yeah, this is one of these "big registers"  - #113 

this is usually done like the `set_model` here
https://github.com/microsoft/jacdac/blob/main/services/modelrunner.md#commands

(you can skip the size, and you would have `pipe command colors { ... }`

**Comment by pelikhan** [comment]
@mmoskal updated

**Review by mmoskal — COMMENTED** [review]
looks good, but I think we need some more info on how the bits are packed (row-major or column-major? are rows/columns aligned to byte/half word/word?)

**Pull request closed without merging**